doc/usecases/plugins/spec/create_array_specification.md

```diff
@@ -0,0 +1,102 @@
+# Use Case: Create array specification for an application
+
+## Summary
+
+- **Scope:** `spec`
+- **Level:** Developer Goal
+- **Actors:** Application, User
+- **Brief:**
+  - Creating a specification for a REST application which needs communication to a mail server and message broker.
```

Comment:
What you give here is an example, not a brief description of the use case.

Reply:
Thanks!